recipes-core/cvm-reverse-proxy-client/cvm-reverse-proxy-client_dev.bb

```diff
@@ -0,0 +1,30 @@
+SUMMARY = "Client Side reverse proxy with confidentialVM attestation support"
+HOMEPAGE = "https://github.com/flashbots/cvm-reverse-proxy"
+LICENSE = "AGPL-3.0-only"
+LIC_FILES_CHKSUM = "file://src/${GO_WORKDIR}/LICENSE;md5=4ae09d45eac4aa08d013b5f2e01c67f6"
+
+inherit go-mod update-rc.d
+
+INITSCRIPT_NAME = "cvm-reverse-proxy-client-init"
+INITSCRIPT_PARAMS = "defaults 85"
+
+GO_IMPORT = "github.com/flashbots/cvm-reverse-proxy"
```

Comment:
where is the version pinned?

Reply:
This is currently a dev instance which is taking the up-to-date version of the cvm-reverse-proxy.
See line 14, `SRCREV = "${AUTOREV}"`. 
If we have a fixed version, we should change SRCREV to point to that exact tag/commit hash